src/Features/DiagnosticsTestUtilities/Diagnostics/AbstractUserDiagnosticTest_NoEditor.cs

```diff
@@ -102,24 +103,21 @@ internal override Task<CodeRefactoring> GetCodeRefactoringAsync(TestWorkspace wo
 
         protected static void AddAnalyzerToWorkspace(Workspace workspace, DiagnosticAnalyzer analyzer, TestParameters parameters)
```

Comment:
no clue why this method is duplicated.  not interested in fixing that right now :)